app/src/androidTest/java/com/github/multimatum_team/multimatum/CalendarActivityTest.kt

```diff
@@ -38,6 +45,30 @@ class CalendarActivityTest {
             .check(ViewAssertions.matches(ViewMatchers.isClickable()))
     }
 
+    @Test
+    fun deadlineCanBeAddedUsingButton() {
+        Espresso.onView(ViewMatchers.withId(R.id.textInputEditCalendar))
+            .perform(click())
+            .perform(typeText("deadlineTestCase"))
+            .perform(ViewActions.closeSoftKeyboard())
+        Espresso.onView(ViewMatchers.withId(R.id.calendar_add_deadline_button))
+            .perform(click())
+        Espresso.onView(ViewMatchers.withId(R.id.textInputEditCalendar))
+            .check(ViewAssertions.matches(ViewMatchers.isClickable()))
+    }
+
+    /*
+    @Test
+    fun keyboardDisappearWhenTouchedOutside() {
```

Comment:
Why this test is in a comment ?

Reply:
It was a tentative for another UI test, I removed it because it is far more complex than I thought. It will be done in the coming weeks.